airbyte_cdk/test/entrypoint_wrapper.py

```diff
@@ -298,10 +298,10 @@ def from_completed_process(
         cls,
         completed_process: subprocess.CompletedProcess[str],
         *,
-        message_file: Path | None = None,
+        stdout_file: Path | None = None,
     ) -> "EntrypointOutput":
-        """Create EntrypointOutput from a completed subprocess with optional message file."""
-        instance = cls(message_file=message_file)
+        """Create EntrypointOutput from a completed subprocess with optional stdout file."""
+        instance = cls(message_file=stdout_file)
```

Comment:
If stdout_file is none, then `completed_process.stdout` might have valid output. Since `stdout_file` is optional, we should fall back to a "captured stdout" scenario, where `stdout` was captured in memory, and in that case would be available in the completed_process object.